gamestonk_terminal/cryptocurrency/discovery/discovery_controller.py

```diff
@@ -0,0 +1,232 @@
+"""Cryptocurrency Overview Controller"""
+__docformat__ = "numpy"
+# pylint: disable=R0904, C0302, W0622
+import argparse
+import os
+from prompt_toolkit.completion import NestedCompleter
+from gamestonk_terminal import feature_flags as gtff
+from gamestonk_terminal.helper_funcs import get_flair
+from gamestonk_terminal.menu import session
+from gamestonk_terminal.cryptocurrency.discovery import (
+    pycoingecko_view,
+    coinpaprika_view,
+    coinmarketcap_view,
+    binance_view,
+)
+
+
+class Controller:
+
+    CHOICES = [
+        "?",
+        "cls",
+        "help",
+        "q",
+        "quit",
+        "cp_search",
+        "cp_coins",
+        "cmc_top",
+        "cg_trending",
+        "cg_most_voted",
+        "cg_most_visited",
+        "cg_top_volume",
+        "cg_recently",
+        "cg_sentiment",
+        "cg_gainers",
+        "cg_losers",
+        "cg_yfarms",
+        "cg_top_defi",
+        "cg_top_dex",
+        "cg_top_nft",
+        "cg_coins",
+        "bin_coins",
+    ]
+
+    def __init__(self):
+        """CONSTRUCTOR"""
+
+        self._discovery_parser = argparse.ArgumentParser(add_help=False, prog="disc")
+        self._discovery_parser.add_argument("cmd", choices=self.CHOICES)
+
+    def print_help(self):
+        """Print help"""
+        print("\nDiscovery:")
+        print("   cls             clear screen")
+        print("   ?/help          show this menu again")
+        print("   q               quit this menu, and shows back to main menu")
+        print("   quit            quit to abandon program")
+        print("")
+
+        print("CoinGecko:")
+        print("   cg_coins           coins available on CoinGecko")
+        print("   cg_trending        trending coins on CoinGecko")
+        print("   cg_most_voted      most voted coins on CoinGecko")
+        print("   cg_most_visited    most visited coins on CoinGecko")
+        print("   cg_top_volume      coins with highest volume on CoinGecko")
+        print("   cg_recently        recently added on CoinGecko")
+        print("   cg_sentiment       coins with most positive sentiment")
+        print(
+            "   cg_gainers         top gainers - coins which price gained the most in given period "
+        )
+        print(
+            "   cg_losers          top losers - coins which price dropped the most in given period "
+        )
+        print("   cg_yfarms          top yield farms")
+        print("   cg_top_defi        top defi protocols")
+        print("   cg_top_dex         top decentralized exchanges")
+        print("   cg_top_nft         top non fungible tokens")
+        print("")
```

Comment:
Since these are becoming bigger, we can remove the print("") in between of the data sources